ovh/data_cloud_project_database_mongodb_user.go

```diff
@@ -54,36 +55,39 @@ func dataSourceCloudProjectDatabaseMongodbUser() *schema.Resource {
 func dataSourceCloudProjectDatabaseMongodbUserRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*Config)
 	serviceName := d.Get("service_name").(string)
-	clusterId := d.Get("cluster_id").(string)
+	clusterID := d.Get("cluster_id").(string)
 
 	listEndpoint := fmt.Sprintf("/cloud/project/%s/database/mongodb/%s/user",
 		url.PathEscape(serviceName),
-		url.PathEscape(clusterId),
+		url.PathEscape(clusterID),
 	)
 
 	listRes := make([]string, 0)
 
-	log.Printf("[DEBUG] Will read users from cluster %s from project %s", clusterId, serviceName)
+	log.Printf("[DEBUG] Will read users from cluster %s from project %s", clusterID, serviceName)
 	if err := config.OVHClient.GetWithContext(ctx, listEndpoint, &listRes); err != nil {
 		return diag.Errorf("Error calling GET %s:\n\t %q", listEndpoint, err)
 	}
 
 	name := d.Get("name").(string)
+	if name == "admin" {
```

Comment:
can you really get there considering this line: https://github.com/ovh/terraform-provider-ovh/pull/609/files#diff-914dfbdacc3d33ba986c5cc7c2ec9931d84789eb769aec8102c606e0a82d07c5R31 ?

Reply:
Yes because the special case for user admin in `ovh/resource_cloud_project_database_mongodb_user.go` will call `ovh/data_cloud_project_database_mongodb_user.go:dataSourceCloudProjectDatabaseMongodbUserRead()` with name = "admin"